chrome/manifest.json

```diff
@@ -40,10 +40,10 @@
   "incognito": "spanning",
   "commands": {
     "_execute_action": {
-      "suggested_key": {
-        "default": "Ctrl+Shift+K",
-        "mac": "Command+Shift+K"
-      }
+      "description": "Open the Kagi extension"
+    },
+    "summarize-active-page": {
+      "description": "Summarize the currently active page"
```

Comment:
You can now set a new shortcut to instantly summarize a page, in your browser settings!